.travis.yml

```diff
@@ -15,9 +15,28 @@
 #
 # Travis CI build file for Swift gRPC.
 
-os:
-  - linux
-  - osx
+matrix:
+  include:
+    - os: linux
+      osx_image: xcode10
```

Comment:
Can `osx_image: xcode10` be omitted for the Linux runs?

Reply:
Done